gslb/ingestion/fullsync.go

```diff
@@ -201,22 +203,42 @@ func checkGDPsAndInitialize() error {
 	return nil
 }
 
-func bootupSync(ctrlList []*GSLBMemberController, gsCache *avicache.AviCache) {
-	gslbutils.Logf("Starting boot up sync, will sync all ingresses, routes and services from all member clusters")
-
-	// add a GDP object
-	err := checkGDPsAndInitialize()
-	if err != nil {
-		// Undefined state, panic
-		panic(err.Error())
+func resyncMemberCluster() {
+	if len(pendingClusters) == 0 {
+		gslbutils.Debugf("Skipping ResyncMemberCluster, no cluster pending")
+		return
 	}
 
-	err = checkGslbHostRulesAndInitialize()
-	if err != nil {
-		// Undefined state, panic
-		panic(err.Error())
+	gslbutils.Logf("Starting cluster sync, will sync objects from pending member clusters: %v", pendingClusters)
+	clients := make(map[string]*kubernetes.Clientset)
+	aviCtrlList := make([]*GSLBMemberController, 0)
+	for cluster := range pendingClusters {
+		gslbutils.Logf("cluster: %s, msg: %s", cluster.clusterName, "initializing")
+		cfg, err := BuildContextConfig(cluster.kubeconfig, cluster.clusterName)
+		if err != nil {
+			gslbutils.Warnf("cluster: %s, msg: %s, %s", cluster.clusterName, "error in connecting to kubernetes API",
+				err)
+			continue
+		} else {
```

Comment:
No need for `else`, we already have a `continue` before this.

Reply:
Done.